### PR TITLE
Improve code style around pypdfium2 usage

### PR DESCRIPTION
**Pull request by mara004**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -214,15 +214,14 @@ def submit_assignment(current_user):
...
         file.save(filename_abs)    
         pdf = pdfium.PdfDocument(filename_abs)
-        n_pages = len(pdf)  # get the number of pages in the document
-        print("no_pages",n_pages)
-        page_indices = [i for i in range(n_pages)]  # all pages
-        renderer = pdf.render(pdfium.PdfBitmap.to_pil,page_indices = page_indices,scale = 300/72)
+        n_pages = len(pdf)
+        print("no_pages", n_pages)
+        renderer = pdf.render(pdfium.PdfBitmap.to_pil, scale=300/72)
         photos_dir =app.config['UPLOAD_FOLDER']+'/'+ str(test_id)+"/"+str(submitted_by)
```
Comment (mara004) [inline app.py:220]: @sakthi1307

Is this regular path joining referring to the host platform?
If so, then you should use `os.path.join()` (or even better, switch to pathlib and join by operator).

Apart from that, for what you currently do, an f-string would be *much* more convenient than cast & concatenate - like this:
```python
photos_dir = app.config['UPLOAD_FOLDER'] + f"/{test_id}/{submitted_by}"
```

But the whole code seems to be a bit of a mess in any case...

Comment (sakthi1307) [reply app.py:220]: Is it normal to store user uploaded data in the host platform ? I was thinking about switching over to Firebase. Except for login works how the overall code can be improved ?

**Review by sakthi1307 — APPROVED** [review]

**Comment by sakthi1307** [comment]
improved submit assignment

File: app.py (hunk 1 above)
Comment (sakthi1307) [reply app.py:220]: apart from how login works, how do i improve the overall code ?

**Pull request merged**